### PR TITLE
CLDR-14634 Adjust Hebrew plural rules

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-14634

Make changes as per ticket, dropping many and adjusting one.

- [ ] This PR completes the ticket.

### CODE REVIEW

File: common/supplemental/plurals.xml [hunk 1]
```diff
@@ -155,9 +155,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
         <!-- 4: one,two,many,other -->
```
Comment (srl295) [inline common/supplemental/plurals.xml:155]:
```suggestion
        <!-- 4: one,two,other -->
```

File: common/supplemental/plurals.xml [hunk 2]
```diff
@@ -155,9 +155,9 @@ CLDR data files are interpreted according to the LDML specification (http://unic
...
 
         <pluralRules locales="he iw">
-            <pluralRule count="one">i = 1 and v = 0 @integer 1</pluralRule>
+            <pluralRule count="one">i = 1 and v = 0 or i = 0 and v != 0 @integer 1</pluralRule>
             <pluralRule count="two">i = 2 and v = 0 @integer 2</pluralRule>
-            <pluralRule count="many">v = 0 and n != 0..10 and n % 10 = 0 @integer 20, 30, 40, 50, 60, 70, 80, 90, 100, 1000, 10000, 100000, 1000000, …</pluralRule>
+            <!--  <pluralRule count="many">v = 0 and n != 0..10 and n % 10 = 0 @integer 20, 30, 40, 50, 60, 70, 80, 90, 100, 1000, 10000, 100000, 1000000, …</pluralRule> -->
```
Comment (srl295) [inline common/supplemental/plurals.xml:160]:
```suggestion
```

**Review by srl295 — COMMENTED** [review]
Does this intend to apply to Yiddish (iw) also?

File: common/supplemental/plurals.xml (hunk 1 above)
Comment (macchiati) [reply common/supplemental/plurals.xml:155]: After all of the plural rules are done, I'm going to apply the program to normalize the output

**Comment by macchiati** [comment]
> Does this intend to apply to Yiddish (iw) also?

No, that language has Germanic plurals.

File: common/supplemental/plurals.xml (hunk 2 above)
Comment (macchiati) [reply common/supplemental/plurals.xml:160]: When the program normalizes the file, that will disappear; it is just for comparison.

**Comment by srl295** [comment]
> > Does this intend to apply to Yiddish (iw) also?
> 
> No, that language has Germanic plurals.

OK.  Then the plural rules need to be split out to reflect that, as `iw` is affected as the PR stands.

**Comment by macchiati** [comment]
Yiddish has the code 'yi'.

The rules are for Hebrew with either the old or new code for:         <
pluralRules locales=*"he iw"*>
You know that, so I suspect your eye just confused 'iw' with 'yi'.

**Comment by srl295** [comment]
> Yiddish has the code 'yi'. The rules are for Hebrew with either the old or new code for: < pluralRules locales=*"he iw"*> You know that, so I suspect your eye just confused 'iw' with 'yi'.
> […](#)

I think you're right.

**Review by srl295 — APPROVED** [review]

File: common/supplemental/plurals.xml (hunk 2 above)
Comment (AEApple) [inline common/supplemental/plurals.xml:160]: I would like to understand why you are commenting out the plural rule for "many" instead of removing it?

Comment (srl295) [reply common/supplemental/plurals.xml:160]: @macchiati explained that this is for comparison and will be removed by later automated processes.

**Pull request merged**